src/js/actions/export.js

```diff
@@ -119,6 +120,45 @@ define(function (require, exports) {
     };
 
     /**
+     * Get current status of generator
+     * @private
+     * @return {Promise.<boolean>}
+     */
+    var _getGeneratorStatus = function () {
+        return descriptor.playObject(generatorLib.getGeneratorStatus())
+            .then(function (status) {
+                return objUtil.getPath(status, "generatorStatus.generatorStatus") === 1;
+            })
+            .catch(function (e) {
+                log.error("Failed to determine generator status: %s", e.message);
+                return Promise.resolve(false);
```

Comment:
For future reference, you can just return the value here instead of a Promise-boxed value.